packages/manager/modules/manager-pci-common/src/utils/index.ts

```diff
@@ -0,0 +1,34 @@
+import { ApiError } from '@ovh-ux/manager-core-api';
+import { ErrorBannerProps } from '@ovh-ux/manager-react-components';
+import { ErrorResponse, isRouteErrorResponse } from 'react-router-dom';
+import { AxiosError } from 'axios';
+
+type RouterErrorResponse = Pick<
+  ErrorResponse,
+  'data' | 'status' | 'statusText'
+> & {
+  internal: boolean;
+};
+
+export const isApiErrorResponse = (error: unknown): error is ApiError =>
+  error instanceof AxiosError &&
```

Comment:
I think it is better to use isAxiosError from axios instead

Reply:
Yes, good catch you're absolutely right, I'll make the change 👍🏻